packages/hemera-mongo-store/test/options.spec.js

```diff
@@ -0,0 +1,238 @@
+'use strict'
+
+const Code = require('code')
+const Utils = require('./utils')
+const expect = Code.expect
+
+describe('Hemera-mongo-store options', function () {
+  const topic = 'mongo-store'
+  const testCollection = 'test'
+  const options = {
+    mongo: {
+      url: 'mongodb://localhost:27017/test'
+    },
+    store: {
+      create: { serializeFunctions : true }, // MF: Dunno how to test this one
+      update: { returnOriginal: false },
+      updateById: { returnOriginal: false },
+      find: { },
+      findById: { explain: true },
```

Comment:
These are the options I'm testing as a sample, not plugin defaults

Here I'm testing 'random' options with results that I can check